cmd/agent/subcommands/snmp/command.go

```diff
@@ -135,7 +135,7 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 				fx.Supply(core.BundleParams{
 					ConfigParams: config.NewAgentParams(globalParams.ConfFilePath, config.WithExtraConfFiles(globalParams.ExtraConfFilePath), config.WithFleetPoliciesDirPath(globalParams.FleetPoliciesDirPath)),
 					SecretParams: secrets.NewEnabledParams(),
-					LogParams:    log.ForOneShot(command.LoggerName, "info", true)}),
+					LogParams:    log.ForOneShot(command.LoggerName, "off", true)}),
```

Comment:
this was changed from `off` to `info` in #29148 , should probably check with @FlorianVeaux for the reasons behind this

Reply:
👀  I think this is likely a testing oversight. I don't think we leverage the logger for printing anything to the console in this command

Reply:
If we don't set it to off the output of `snmp walk` is poluted by logs like this unfortunately
```
2024-11-07 09:39:39 UTC | CORE | INFO | (pkg/aggregator/aggregator.go:762 in run) | Stopping aggregator
```